code/src/components/Game.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import { Start } from "./Start"
+import { GameBoard } from "./GameBoard"
+import { LoadingIndicator } from "./LoadingIndicator"
+
+export const Game = () => {
+  const loading = useSelector((store) => store.ui.loading)
```

Comment:
Keeping the loading function separately in the ui component is a reasonable decision as well.